shell/packages/sandstorm-accounts-ui/login_buttons.js

```diff
@@ -5,6 +5,9 @@ var helpers = {
   isDemoUser: function () {
     return this._db.isDemoUser();
   },
+  isAdmin: function () {
+    return Meteor.user().isAdmin;
```

Comment:
Question to e.g. Drew: I notice we're access `this._db` but it seems more idiomatic to access `Meteor.user()`, so I went with what I think is more idiomatic.

Should I do things more like `isDemoUser` and access `this._db` instead?